### DISCUSSION TITLE
DAYNAME based another field date or date-time

### DISCUSSION THREAD

**Discussion by shawkatmarrouf** [YetiForceCompany/YetiForceCRM#17534] [How to]
Is there any idea of how to print out "dayname" in a custom field based on a day-time field like record created time using workflow ?

**Comment by mariuszkrzaczkowski** [comment]
it is possible to add the format attribute to a variable

**Reply by shawkatmarrouf** [reply]
I've used DAYNAME attribute as expression and field in workflow but didn't work, is there an example or guide?

**Reply by mariuszkrzaczkowski** [reply]
where do you want to display it?

**Reply by shawkatmarrouf** [reply]
In a custom text field inside custom module record using workflow "update field" to make that field = "createdtime" and display it in a format of day name

**Reply by shawkatmarrouf** [reply]
any update please
